app/assets/javascripts/headshot_support.js

```diff
@@ -0,0 +1,27 @@
+var headshotUtils = {
+    getPageMetaTags:function () {
+        return document.getElementsByTagName('meta');
+    },
+    getPageMetaValue:function (metaName) {
+        var metaValue = '';
+        var pageMetaTags = headshotUtils.getPageMetaTags();
+        if (pageMetaTags.length > 0) {
+            for (var i = 0; i < pageMetaTags.length; i++) {
+                if (pageMetaTags[i]['name'] == metaName) {
+                    metaValue = pageMetaTags[i]['content'];
+                }
+            }
+        }
+
+        return metaValue;
+    }
+};
+
+var headshotInitializer = {
+    setup: function(width, height, railsTokens) {
+        document.write(headshot.get_html(width, height, railsTokens));
```

Comment:
document.write can be a form of eval.<br>Identifier 'get_html' is not in camel case.<br>'headshot' is not defined.